projects/ad713x_fmcz/Makefile

```diff
@@ -0,0 +1,6 @@
+TARGET := ad713x_fmcz
```

Comment:
Curiosity: Shouldn't TINYIIOD be added here?

Reply:
No, this Makefile builds `ad713x_fmcz` project. For building project with iio you can run `make TINYIIOD=y`